apps/fabric-website/src/pages/Overviews/ResourcesPage/docs/default/ResourcesDeveloperResources.md

```diff
@@ -7,6 +7,14 @@ Get started with React and learn how to build your first projects.
   <li class="mdut--full">[Fabric React tutorial](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Getting-Started-with-UI-Fabric)</li>
 </ul>
 
+### Release notes and demos
+
+View Fabric's versioned release notes and component demos.
```

Comment:
Since this page is for Fabric on all platforms, probably good to specify Fabric React here.
```suggestion
View Fabric React's versioned release notes and component demos.
```

Reply:
Right! Great suggestion @ecraig12345.